<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USDB Syncer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,10,2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>347</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Synchronize to local song directory</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_song_dir">
           <property name="text">
            <string>./songs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_select_song_dir">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Select song folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="lineEdit_search">
        <property name="placeholderText">
         <string>Search for anything ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_filters">
        <property name="text">
         <string>Clear filters</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Filters</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,2,0,2,0,2,0,2,0,1,0,1,0,1,0">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/artist.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_artist">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/title.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_title">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/language.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_language">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/edition.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_edition">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/golden_notes.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_golden_notes"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/rating.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_rating"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/views.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_views"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_sync_status"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView_availableSongs">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>24</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="toolButton_infos">
             <property name="toolTip">
              <string>Information</string>
             </property>
             <property name="text">
              <string>Infos</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_warnings">
             <property name="toolTip">
              <string>Warnings</string>
             </property>
             <property name="text">
              <string>Warnings</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/warning.png</normaloff>:/icons/warning.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_errors">
             <property name="toolTip">
              <string>Errors</string>
             </property>
             <property name="text">
              <string>Errors</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/error.png</normaloff>:/icons/error.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_downloadSelectedSongs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download selected songs!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_settings"/>
    <addaction name="action_meta_tags"/>
    <addaction name="action_generate_song_pdf"/>
    <addaction name="action_import"/>
   </widget>
   <widget class="QMenu" name="menu_sync">
    <property name="title">
     <string>Synchronize</string>
    </property>
    <addaction name="action_refetch_song_list"/>
    <addaction name="action_select_local_songs"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_show_log"/>
   </widget>
   <addaction name="menu_sync"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_meta_tags">
   <property name="text">
    <string>Create Meta Tags</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_generate_song_pdf">
   <property name="text">
    <string>Generate Song List (PDF)...</string>
   </property>
  </action>
  <action name="action_refetch_song_list">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/faviconUSDB.ico</normaloff>:/icons/faviconUSDB.ico</iconset>
   </property>
   <property name="text">
    <string>Check USDB Song List</string>
   </property>
  </action>
  <action name="action_select_local_songs">
   <property name="text">
    <string>Select local Songs</string>
   </property>
  </action>
  <action name="action_show_log">
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="action_import">
   <property name="text">
    <string>Import...</string>
   </property>
   <property name="toolTip">
    <string>Importiere USDB IDs</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
